database/postgres/repo_list.go

```diff
@@ -55,8 +55,50 @@ func (c *client) GetRepoList() ([]*library.Repo, error) {
 	return repos, nil
 }
 
+// GetOrgPrivateRepoList gets a list of all private repos by org from the database.
+func (c *client) GetOrgPrivateRepoList(org string) ([]*library.Repo, error) {
```

Comment:
Think we're missing a couple parameters here for pagination (`page` and `perPage`):

```suggestion
func (c *client) GetOrgPrivateRepoList(org string, page, perPage int) ([]*library.Repo, error) {
```